apps/s_server.c

```diff
@@ -2328,6 +2328,30 @@ static void print_stats(BIO *bio, SSL_CTX *ssl_ctx)
                SSL_CTX_sess_get_cache_size(ssl_ctx));
 }
 
+static long count_reads_callback(BIO *bio, int cmd, const char *argp, size_t len,
+                                 int argi, long argl, int ret, size_t *processed)
+{
+    unsigned *p_counter = (unsigned*)BIO_get_callback_arg(bio);
```

Comment:
`(unsigned *)`

Reply:
Please make this `unsigned int *` rather than just `unsigned *`